pyanaconda/installation.py

```diff
@@ -380,3 +380,5 @@ def run_installation(payload, ksdata):
 
     # done
     progress_complete()
+    log.info("All tasks in the installation queue are done. Installation successfully finished.")
+    # this message is automatically detected by QE tools, do not change it lightly
```

Comment:
Please move the comment above the line.

By the prefix or postfix (which we don't have right now), we could make this more obvious than having a comment. 
Just to explain what I'm thinking:
 ```
log.info("REPORT: All tasks in the installation queue....")
```
The upper `REPORT` will give you a hint that the placeholder is used for something. Ideally it should be explained also in the documentation of course.

However, I don't mind any of these solutions.

Reply:
I guess we could have a special logger for that. It is definitely an interesting idea, but it needs to be investigated and discussed with the team and QA. That seems to be out of scope of this issue.

Reply:
IMO, do not make it more complicated than it has to be. All we care about is a message (in syslog/journal) that would be sent via the virtio port to the host system.